reference_material/data_dictionary

```diff
@@ -0,0 +1,130 @@
+# Current available fields:
+
+## api_endpoint_id:
+The unique identifier for the API endpoint.
+Type: numeric
+## api_id:
+The ID of the API.
+Type: numeric
+
+## api_vendor_id:
+The ID of the API vendor.
+Type: numeric
+
+## api_vendor
+The vendor of the API, e.g "OneLook", "Victoria State Government Australia", "TeejLab"...
+Type: string
+
+## api:
+The name of the API, e.g "Datamuse Dictionary", "DepartmentAPI"...
+Type: string
+
+## category
+The category of the API, e.g "Research & Education", "AI & Data Science"...
+Type: string
+
+## usage_base:
+Free or commercial API.
+Type: string
+
+## sample_response:
+A sample response from the API.
+Type: JSON
+
+## tagset:
+List of keys in the sample response.
+Type: list
+
+## authentication:
+Authentication method, e.g "None", "OAuth 2.0", "Basic Authentication"...
+Type: string
+
+## security_test_category:
+The category of the security test, e.g Broken Authentication', 'Cross-Site Scripting', 'Insecure Deserialization', 'SQL Injection','XML External Entities', 'Buffer Overflow'....
+Type: string
+
+## security_test_result:
+The result of the security test, e.g (FALSE=Passed; TRUE=Failed)
+Type: boolean
+
+## server_location:
+The location of the API, e.g "Australia", "United States"...
+Type: string
+
+## hosting_isp:
+The hosting ISP of the API, e.g "Amazon", "Cloudflare", "Google"...
+Type: string
+
+## server_name:
```

Comment:
If the server name is exposed, there is a risk. Otherwise, low risk

Reply:
I agree, but any of the fields exposed are risks, not just the server name. They called it vector attack